Cargo.toml

```diff
@@ -0,0 +1,95 @@
+[package]
+name = "zinc"
+version = "0.1.0"
+authors = ["Zinc Developers <zinc@github.com>"]
+build = "build.rs"
+
+[lib]
+name = "zinc"
+crate-type = ["lib"]
```

Comment:
There was also a bug for dylib not working in this case that is fixed in cargo upstream: https://github.com/rust-lang/cargo/issues/1612.  That being said, rlib is the proper thing to have here.